### PR TITLE
Update velocity egg

### PR DESCRIPTION
**Pull request by xXTheBigBearXx**
# Description

- Updates velocity.toml to the latest version, and changes any 127.0.0.1 entries to 172.18.0.1 (The default IP for pterodactyl0)

- Removes some text in the "Velocity Version Variable" referring to '1.1.x' as the latest dev version, as since Velocity merged into PaperMC the latest dev build is the same as the latest version

## Checklist for all submissions

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/parkervcp/eggs/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [x] Have you tested and reviewed your changes with confidence that everything works?
* [x] Did you branch your changes and PR from that branch and not from your master branch?
  * If not, why?:

* [x] You verify that the start command applied does not use a shell script
  * [x] If some script is needed then it is part of a current yolk or a PR to add one
* [x] The egg was exported from the panel

### CODE REVIEW

**Comment by parkervcp** [comment]
pterodactyl already replaces 127.0.0.1 with the ip address assigned to pterodactyl_nw. If anything this would break that and potentially break anyone who isn;'t using 172.18.0.1

File: game_eggs/minecraft/proxy/java/velocity/velocity.toml [hunk 1]
```diff
@@ -72,25 +72,25 @@ enable-player-address-logging = true
 [servers]
 # Configure your servers here. Each key represents the server's name, and the value
 # represents the IP address of the server to connect to.
-lobby = "127.0.0.1:30066"
-factions = "127.0.0.1:30067"
-minigames = "127.0.0.1:30068"
+lobby = "172.18.0.1:30066"
```
Comment (parkervcp) [inline game_eggs/minecraft/proxy/java/velocity/velocity.toml:75]: This will break the panel updating the address on its own.

**Review by parkervcp — CHANGES_REQUESTED** [review]

**Comment by xXTheBigBearXx** [comment]
How does it perform that change?
From testing I could not connect to a server with a 127.0.0.1 IP assigned, with 127.0.0.1 in the .toml, but it would connect as soon as I changed it to 172.18.0.1 in the .toml

File: game_eggs/minecraft/proxy/java/velocity/velocity.toml (hunk 1 above)
Comment (xXTheBigBearXx) [reply game_eggs/minecraft/proxy/java/velocity/velocity.toml]: Reverted to 127.0.0.1

**Comment by parkervcp** [comment]
The panel updated the config on startup replacing the 127.0.0.1 in the toml on its own.

**Comment by xXTheBigBearXx** [comment]
> on startup

Of the Container? If so it has never done so for me - is it something that needs enabling or should it be active by default?

**Comment by parkervcp** [comment]
it doesn't touch the toml. I know it will update the server properties and the yaml for waterfall/bungeecord I believe.

**Comment by QuintenQVD0** [comment]
You where right changing this to 172.18.0.1, the configuration parser does indeed not touch it.

**Review by parkervcp — APPROVED** [review]

**Pull request merged**